### PR TITLE
Implement an ETS storage adapter

### PR DESCRIPTION
**Pull request by doomspork**
This requires #129 is merged beforehand but this leverages those new changes and implements and ETS adapter in place for Ecto/DB storage.

cc @alexfilatov

### CODE REVIEW

File: lib/guardian/db/adapter/ets.ex [hunk 1]
```diff
@@ -0,0 +1,88 @@
+defmodule Guardian.DB.ETSAdapter do
+  @moduledoc """
+  Implement the Guardian.DB.Adapter for ETS
```
Comment (doomspork) [inline lib/guardian/db/adapter/ets.ex]: If everyone is :+1: on adding this adapter I'll expand on the module doc to talk about creating the table etc.

Comment (alexfilatov) [reply lib/guardian/db/adapter/ets.ex]: 👍as for me

File: lib/guardian/db/adapter/ets.ex [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+  end
+
+  @impl true
+  def delete(%{jti: jti} = token, opts) do
+    table = Keyword.fetch!(opts, :table)
+
+    if :ets.delete(table, jti) do
+      {:ok, token}
+    else
+      {:error, token}
```
Comment (aleDsz) [inline lib/guardian/db/adapter/ets.ex]: What about adding an error message here? Even if we don't have the reason from `:ets`

**Review by aleDsz — COMMENTED** [review]
Just one little adjustment, if u think it makes sense :)

File: lib/guardian/db/adapter/ets.ex (hunk 2 above)
Comment (doomspork) [reply lib/guardian/db/adapter/ets.ex]: Any suggestions for this @aleDsz?

**Comment by yordis** [comment]
@doomspork any updates? Would you like me to take ownership of it? If so, what is left?

**Comment by doomspork** [comment]
@yordis feel free to take this over and drive it across the finish line. I was waiting for an answer above on the error message but we should just make a sane decision and move forward.

**Comment by alexfilatov** [comment]
please let me know if I can help with anything @doomspork @yordis

**Comment by doomspork** [comment]
Thank you @alexfilatov! Would you mind giving this PR another thorough look through and review?

File: lib/guardian/db/adapter/ets.ex (hunk 2 above)
Comment (aleDsz) [reply lib/guardian/db/adapter/ets.ex]: Sorry for the delay, what about `{:error, "Failed to delete from ETS table"}`?

**Comment by yordis** [comment]
I am gonna be working over the weekend, but please feel free to take the lead for sure!

**Comment by alexfilatov** [comment]
the PR looks good to me, I can only approve here in the comments :)
@yordis @aleDsz what do you think?

**Comment by yordis** [comment]
@alexfilatov working on it at the very moment!

**Comment by yordis** [comment]
@alexfilatov do you mind giving a +1, please verify latest version if you could

**Comment by yordis** [comment]
Looking at the code, we need to improve it by returning the `changeset` with an error in some cases!

**Review by alexfilatov — APPROVED** [review]
Looks good!

**Review by yordis — APPROVED** [review]

**Pull request merged**